packages/server/graphql/mutations/downgradeToStarter.ts

```diff
@@ -42,6 +48,29 @@ export default {
     }
 
     // RESOLUTION
+    if (!isSuperUser) {
+      const [organizationUsers, organization] = await Promise.all([
+        dataLoader.get('organizationUsersByOrgId').load(orgId),
+        dataLoader.get('organizations').load(orgId)
+      ])
+      const billingLeaderUserId = organizationUsers
+        .filter((organizationUser) => organizationUser.role === 'BILLING_LEADER')
+        .find(({userId}) => userId)!.id
+      const billingLeaderUser = await dataLoader.get('users').load(billingLeaderUserId)
+      if (!billingLeaderUser) {
+        const error = new Error('Unable to find billing leader')
+        sendToSentry(error, {userId: viewerId})
+        return
+      }
+      analytics.organizationDowngraded(viewerId, {
+        orgId,
+        oldTier: tier,
+        newTier: 'starter',
+        billingLeaderEmail: billingLeaderUser.email,
```

Comment:
@tghanken `organizationDowngraded` accepts only one billing leader email string, but there could be many billing leaders in an org. I've grabbed the first billing leader. Does that work for you or would you like all the billing leader emails?